packages/valory/skills/market_creation_manager_abci/models.py

```diff
@@ -80,6 +84,24 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.max_proposed_markets = kwargs.get(
             "max_proposed_markets", DEFAULT_MAX_PROPOSED_MARKETS
         )
+        self.max_approved_markets = kwargs.get(
+            "max_approved_markets", DEFAULT_MAX_APPROVED_MARKETS
+        )
+        self.markets_to_approve_per_epoch = kwargs.get(
+            "markets_to_approve_per_epoch", DEFAULT_MARKETS_TO_APPROVE_PER_EPOCH
+        )
+        self.min_approve_markets_epoch_seconds = kwargs.get(
+            "min_approve_markets_epoch_seconds",
+            DEFAULT_MIN_APPROVE_MARKETS_EPOCH_SECONDS,
+        )
+        self.approve_market_event_days_offset = kwargs.get(
+            "approve_market_event_days_offset", DEFAULT_APPROVE_MARKET_EVENT_DAYS_OFFSET
+        )
```

Comment:
Why use defaults instead of `_ensure`? The defaults are already in the config file.

Reply:
This is how it has been done for other variables. It took the same methodology of other services. There are not a established procedure or recommendation for this. What do you suggest?

Reply:
The suggested way is to always use the `_ensure`.

Reply:
fixed